### DISCUSSION TITLE
Aux Lüfter P1P mit upgrade Kit auf P1S starten?

### DISCUSSION THREAD

**Discussion by Lollo76** [OrcaSlicer/OrcaSlicer#1704] [Q&A]
Hallo, wie kann ich silcer einstellen das der aux lüfter (der im bauraum) beim druck starten soll, ich kann ihn zwa so anstellen, aber er geht nicht automatisch an, kann auch nicht die geschwindigkeit unter filament ändern, jemand ne idee?

danke

**Comment by SurgeHawk** [comment]
Under filament settings you will find the cooling tab. You can set the aux fan speed there. Hope this helps.
